packages/contracts/test/diamond/facets/ChefFacet.t.sol

```diff
@@ -294,7 +286,7 @@ contract DepositStateChefTest is DepositStateChef {
         blocks = bound(blocks, 45361, 2 ** 128 - 1);
         assertEq(IChefFacet.totalShares(), shares);
 
-        uint256 preBal = governanceToken.balanceOf(fourthAccount);
+        uint256 preBal = IGovToken.balanceOf(fourthAccount);
```

Comment:
why you are renaming instances as interfaces?

Reply:
my mistake, starting to understand things better now. Cheers Paul

Reply:
I see, but make it work with inheritance from a helper file